subsys/bluetooth/host/att.c

```diff
@@ -139,6 +139,8 @@ static k_tid_t att_handle_rsp_thread;
 
 struct bt_att_tx_meta_data {
 	struct bt_att_chan *att_chan;
+	bt_gatt_multi_func_t multi_func;
+	uint16_t num_attr;
 	bt_gatt_complete_func_t func;
```

Comment:
This can be `union`ed. Discriminated on `num_attr == 0` or maybe `== 1`. 
```suggestion
	uint16_t num_attr;
	union {
		bt_gatt_complete_func_t func;  /* Active when `num_attr == 0`.*/
		bt_gatt_multi_func_t multi_func;  /* Active when `num_attr != 0` */
	}
```